to be continued

seehase opened the pull request:
New bootstrap navbar menu
Cleaned Color thems
reworked styles
...

seehase committed:
Fixed version check

seehase committed:
fixed usage of $txt -> use getText()

seehase committed:
Removed files from /styles/images and fonts (can not see that we use them and if put it in our image folder)
Change index to user color from theme as card background

seehase committed:
Cleaned and formatted  index_js

seehase committed:
Cards on indexpage can be configured (which and order)

[layout]
cards = "day, month, year, allYears, cumulative, yearPerMonth,  info, image, top"

I'll fix the missing images later :-(

seehase committed:
Fixed missing images... all fine now

seehase committed:
small changes and use day_chart_utc

seehase committed:
fixed the double slash issue :-(

seehase committed:
small adjustments

seehase committed:
wip

seehase committed:
Reworked version check... not sure if we want to keep this "blinker" so prominent in the footer? any proposal?
to test... change temporary version info to $version = "v2023.06.21 (dev)"; e.g. and delete session

seehase committed:
Reworked generic importer include ... validation will come, until then if no or nor valid parameter defined -> no error

ericplan committed:
Add files via upload

Refactored logo.

ericplan committed:
Update topmenu.php

Replaced logo with .svg version. Removed $ligado (not in use as far as I know)

ericplan committed:
Update index_charts.js

Repairing top31 page

ericplan committed:
Update top31_chart.php

This is the correct chart which reorders on selection. Sorry if I introduced something you changed an hour or so. To tired to check it.

ericplan committed:
Update style.css

Minor changes to the side _menu

ericplan committed:
Update style.css

Temporary style adjustments

seehase committed:
Fix top31 groupby and isset check

seehase committed:
Removed blinker and corrected top31 statement, "geg_maand" is not needed at all in the select, only the sum(geg_maand)
Please check again

seehase committed:
Fix TopMenu

ericplan committed:
Update top31.php

corrected hideFooter

seehausen commented on ZonPHP/pages/top31.php:
```diff
@@ -34,7 +34,7 @@
 }
 $footer_display_style = "clear:both; ";
-if (isset($params['hideFooter'])) {
+if ($params['hideFooter'] == true) {
```
THANKS... took me 1h to find the GUI bug!

I saw we need to add a closing </div> in all pages, I will add that later

seehase committed:
Fixed HTML validation errors

seehase committed:
missed one

seehase committed:
fix weewx path

seehase committed:
Reponsive Header/footer

seehase committed:
Changed taal -> Language request parameter

seehase committed:
Enables single card layout

ericplan committed:
Add files via upload

Slightly changed logo with text left to the symbol

ericplan committed:
Possible themes logo

Themes logo in svg, free to use, could be changed a bit when wanted
https://uxwing.com/themes-icon

ericplan committed:
Add files via upload

This svg is easy to adapt to our needs. This version has the same 4x3 size as the flags

seehase committed:
Reworked Menu

seehase committed:
Cleanup styles ... t.b.c.

seehase committed:
Changes color handling, added "#" in the color themes and replaces <?php  --> <?=

seehase committed:
replaced all <?php echo

seehase committed:
Small ajdustments

seehase committed:
TopMenu

seehase committed:
small hack

ericplan committed:
Rename themes 4x3.svg to themes_4x3.svg

No spaces in filenames

seehase committed:
Fix footer

seehase committed:
Fixed color rework

seehase committed:
different sessionID for multi instance

seehase committed:
shop_plant update

seehase committed:
HTML validation fixes

seehase committed:
Update VersionInfo
